addons/l10n_ph/views/report_invoice.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="l10n_ph_report_invoice_document_inherit" inherit_id="account.report_invoice_document">
+        <xpath expr="//span[@t-field='o.partner_id.vat']" position="attributes">
+            <attribute name="t-if">o.company_id.country_id.code != 'PH' or o.partner_id.branch_code == '000'</attribute>
+        </xpath>
+        <xpath expr="//span[@t-field='o.partner_id.vat']" position="after">
+            <span t-if="o.company_id.country_id.code == 'PH' and o.partner_id.branch_code != '000'" t-esc="'%s-%s' % (o.partner_id.vat, o.partner_id.branch_code)"/>
```

Comment:
If in the future we need to make this stuff (concatenate vat and branch code) and compute non stored field will be useful

Reply:
I wonder if it should not be in the vat number itself instead of bothering with the a branch code field...  The branch code field could be computed...